great_expectations/compatibility/google.py

```diff
@@ -24,7 +24,7 @@
 try:
     from google.auth.exceptions import DefaultCredentialsError
 except (ImportError, AttributeError):
-    DefaultCredentialsError = GOOGLE_CLOUD_STORAGE_NOT_IMPORTED
+    DefaultCredentialsError = GOOGLE_CLOUD_STORAGE_NOT_IMPORTED  # type: ignore[assignment,misc]
```

Comment:
WTH?

Reply:
These were random typing issues that came up when this PR was in CI. Not sure why this happened now but not previously.